src/components/account/UserButtons.vue

```diff
@@ -42,6 +56,29 @@
         >
           <p>{{ $t('set.notifications.likeLimit.body') }}</p>
         </b-modal>
+        <b-modal
+          :id="pinnedLimitModalId"
+          :title="$t('entity.notifications.pinLimit.title')"
+          hide-footer
+          hide-header-close
+        >
+          {{ $t('entity.notifications.pinLimit.body') }}
+          <div class="modal-footer">
+            <b-button
+              variant="outline-primary"
+              data-qa="cancel button"
+              @click="$bvModal.hide(pinnedLimitModalId)"
+            >
+              {{ $t('actions.close') }}
+            </b-button>
+            <b-button
+              variant="primary"
+              @click="viewPins()"
+            >
+              {{ $t('entity.actions.viewPinned') }}
+            </b-button>
+          </div>
+        </b-modal>
```

Comment:
This could possibly be removed and be merged with the new PinModal component, by changing PinModal' description and the labels of its buttons.
I think this code could be avoided here (since there is a modal for the workflow), unlike the one above about likes, where clicking the like button doesn't open a modal normally.